content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst

```diff
@@ -0,0 +1,153 @@
+   warehouse, begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`,
+   then select a warehouse from the :guilabel:`Warehouses` screen.
+
+   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input field
+   and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick components
+   and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then store
+   products (3 steps)`.
+
+   .. image:: two_step_manufacturing/manufacturing-type.png
+      :align: center
+      :alt: The Manufacture radio input field on a warehouse configuration page.
+
+.. important::
+   Products must be properly configured before they can be manufactured in Odoo. For details on how
+   to do so, see the documentation on how to :ref:`configure a product for manufacturing
+   <manufacturing/management/configure-manufacturing-product>`.
+
+Create manufacturing order
+==========================
+
+To manufacture a product in Odoo *Manufacturing*, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then click :guilabel:`New`
+to create a new manufacturing order.
+
+On the new manufacturing order, select the product to be produced from the :guilabel:`Product`
+drop-down menu. The :guilabel:`Bill of Material` field auto-populates with the associated Bill of
+Materials (BoM).
+
+If a product has more than one |BOM| configured for it, the specific |BOM| can be selected in the
+:guilabel:`Bill of Material` field, and the :guilabel:`Product` field auto-populates with the
+associated product.
+
+After a |BOM| has been selected, the :guilabel:`Components` and :guilabel:`Work Orders` tabs
+auto-populate with the components and operations specified on the |BOM|. If additional components or
+operations are required for the manufacturing order being configured, add them to the
+:guilabel:`Components` and :guilabel:`Work Orders` tabs by clicking :guilabel:`Add a line`.
+
+Finally, click :guilabel:`Confirm` to confirm the manufacturing order.
+
+Process pick components transfer
+================================
+
+After confirming a two-step manufacturing order, a :guilabel:`Transfers` smart button appears at the
+top of the page. Click it to open the pick components transfer for the manufacturing order. This
+transfer is used to track the movement of components from the locations where they are stored to the
+location where they are used to manufacture the product.
+
+.. image:: two_step_manufacturing/transfers-smart-button.png
+   :align: center
+   :alt: The Transfers smart button on a manufacturing order.
+
+After transferring the components out of their storage location, click :guilabel:`Validate` at the
+top of the transfer, followed by :guilabel:`Apply` on the :guilabel:`Immediate Transfer?` pop-up
+window that appears. Doing so moves the transfer to the :guilabel:`Done` stage and updates inventory
```

Comment:
I think this is the first instance of this doc talking about "stages". Do you think this is something readers should know at this point? Maybe in the following **Transfers** smart button screenshot, it can also include an image of the Done stage, where you point it out with an arrow? Just a thought!